ts/src/mexc.ts

```diff
@@ -5020,7 +5046,7 @@ export default class mexc extends Exchange {
             if (method === 'POST') {
                 headers['Content-Type'] = 'application/json';
             }
-        } else if (section === 'contract' || section === 'spot2') {
+        } else if (section === 'contract' || section === 'spot2' || section === 'broker') {
```

Comment:
@sc0Vu that is the reason, the brokerAPi should enter the spot branch because `signature` is required in the URL and not in the headers

Reply:
@carlosmiei thanks. Where do you find the signature stuff?

Reply:
@sc0Vu I didn't, it was trial and error 😅